pkg/machinery/config/types/network/default_action_config.go

```diff
@@ -36,6 +38,8 @@ var (
 //	examples:
 //	  - value: exampleDefaultActionConfigV1Alpha1()
 //	alias: NetworkDefaultActionConfig
+//	schemaRoot: true
+//	schemaMeta: v1alpha1/NetworkDefaultActionConfig
```

Comment:
Added these two new fields.
`schemaMeta` - hints docgen that this document has a required "meta" section which should match the apiVersion/Type in the value.
`schemaRoot` - it will end up in the `oneOf` section at the root of the JSON schema. e.g.:
```json
{
  "oneOf": [
    {
      "$ref": "#/$defs/network.DefaultActionConfigV1Alpha1"
    },
    {
      "$ref": "#/$defs/network.RuleConfigV1Alpha1"
    },
    {
      "$ref": "#/$defs/runtime.EventSinkV1Alpha1"
    },
    {
      "$ref": "#/$defs/runtime.KmsgLogV1Alpha1"
    },
    {
      "$ref": "#/$defs/siderolink.ConfigV1Alpha1"
    },
    {
      "$ref": "#/$defs/v1alpha1.Config"
    }
  ]
}